micropy/project/modules/packages.py

```diff
@@ -3,13 +3,14 @@
 """Project Packages Module."""
 
 import shutil
-import tempfile
-from json import JSONDecodeError
 from pathlib import Path
+from typing import Any, Union
 
-import requirements
+from boltons import fileutils
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'boltons'](https://app.codacy.com/app/BradenM/micropy-cli/pullRequest?prid=4593453)